src/Compilers/VisualBasic/Test/Semantic/Semantics/RefFieldTests.vb

```diff
@@ -0,0 +1,41 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Imports Microsoft.CodeAnalysis.Test.Utilities
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.UnitTests
+
+    Public Class RefFieldTests
+        Inherits BasicTestBase
+
+        <Fact>
+        Public Sub RefField()
```

Comment:
>RefField [](http://example.com/codeflow?start=19&length=8)

I am not sure what this test is actually testing. Is the field actually imported since it is private? I would at least expect this test to cover PEFieldSymbol behavior, where custom modifiers go, etc.